.github/workflows/haskell.yml

```diff
@@ -0,0 +1,53 @@
+name: Haskell
+on:
+  push:
+  pull_request:
+
+jobs:
+  ci:
+    strategy:
+      fail-fast: false
+      matrix:
+        ghc: [8.10.7]
+    runs-on: ubuntu-latest
+    container:
+      # ubuntu with ghcup already installed
+      image: ghcr.io/facebookincubator/hsthrift/ci-base:ghcup
+      options: --cpus 2 --security-opt=seccomp=unconfined
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install additional tools
+        run: |
+          apt-get update
+          apt install -y unzip
+        working-directory: bindings/haskell
```

Comment:
nit: Bunch of these `working-directory: bindings/haskell` lines could be removed without changing the semantics.